internal_packages/keybase/stylesheets/main.less

```diff
@@ -0,0 +1,88 @@
+@import "ui-variables";
+@import "ui-mixins";
+
+.keybase-profile {
+  border: 1px solid black;
```

Comment:
Technically, we'll want to pull the appropriate variables for padding, background and border colors from `ui-variables` before we ship this. For example this would probably `border-color-primary`. Can wait for design pass though :-)